integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -166,6 +167,38 @@ def read_parquet_sql(data_path):
 
     assert_gpu_fallback_collect(read_parquet_sql(data_path), "FileSourceScanExec", conf={"spark.rapids.sql.format.parquet.reader.type": "PERFILE"})
 
+@allow_non_gpu("SerializeFromObjectExec", "DeserializeToObjectExec",
+               "FilterExec", "MapElementsExec", "ProjectExec")
+@delta_lake
+@ignore_order
+@pytest.mark.skipif(not supports_delta_lake_deletion_vectors() or not is_spark_353_or_later(), \
+                    reason="Deletion vectors new in Delta Lake 2.4 / Apache Spark 3.4")
+def test_delta_deletion_vector_perfile_read(spark_tmp_path):
```

Comment:
Can we also parameterize this test with different row filter types?

Reply:
That is an excellent comment. I looked into it and it's not a user-config that can be set.

Reply:
OK, I see it can be set with CDC. We will test it later then when we support CDC.